katka/models.py

```diff
@@ -111,3 +111,13 @@ class SCMStepRun(AuditedModel):
     status = models.CharField(max_length=30, choices=STEP_STATUS_CHOICES, default=STEP_STATUS_INPROGRESS)
     output = models.TextField(blank=True)
     scm_pipeline_run = models.ForeignKey(SCMPipelineRun, on_delete=models.PROTECT)
+
+
+# SCM Releases, comprises a range of commits
+class SCMRelease(AuditedModel):
+    public_identifier = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=255)
+    released = models.DateTimeField(null=True)
+    start_commit_hash = models.CharField(max_length=64)
+    end_commit_hash = models.CharField(max_length=64)
+    last_scm_pipeline_run = models.ForeignKey(SCMPipelineRun, on_delete=models.PROTECT)
```

Comment:
Please add a comment to explain this filed and maybe calling it `scm_pipeline_run `